From 67b64a1fdd67007770a52333d137d0f04f0c56ca Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Fri, 19 Jun 2020 14:18:49 +0800
Subject: [PATCH] net: wireless: rockchip_wlan: fix missing MODULE_LICENSE
 warning

WARNING: modpost: missing MODULE_LICENSE() in drivers/net/wireless/rockchip_wlan/rkwifi/rk_wifi_config.o
see include/linux/module.h for more information

Change-Id: Ie683a00a1d0c66633fa7c169179298b08ef95452
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 drivers/net/wireless/rockchip_wlan/rkwifi/rk_wifi_config.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/wireless/rockchip_wlan/rkwifi/rk_wifi_config.c b/drivers/net/wireless/rockchip_wlan/rkwifi/rk_wifi_config.c
index 73e727641638..82799df987e8 100644
--- a/drivers/net/wireless/rockchip_wlan/rkwifi/rk_wifi_config.c
+++ b/drivers/net/wireless/rockchip_wlan/rkwifi/rk_wifi_config.c
@@ -27,3 +27,4 @@ int rkwifi_set_firmware(char *fw, char *nvram)
 	return 0;
 }
 EXPORT_SYMBOL(rkwifi_set_firmware);
+MODULE_LICENSE("GPL");
-- 
2.35.3

